public/app/plugins/datasource/cloudwatch/mocks/cloudwatch-logs-test-data/singleLineFullQuery.ts

```diff
@@ -1,26 +1,3 @@
-import { monacoTypes } from '@grafana/ui';
```

Comment:
Just a suggestion: since these files just contain query strings, maybe we can move them all to one file for single line queries?

Reply:
good idea! i'll probably do that all in one go in the follow up for the refactor of the other languages as well.